### PR TITLE
From REQUIRE to Project.toml

### PR DESCRIPTION
**Pull request by andreasnoack**

### CODE REVIEW

**Comment by andreasnoack** [comment]
I also bumped the version here so we can use this commit for the release

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name = "ForwardDiff"
+uuid = "f6369f11-7733-5829-9624-2563aa707210"
+version = "0.11.0"
```
Comment (fredrikekre) [inline Project.toml]:
```suggestion
version = "0.10.1"
```

Comment (KristofferC) [reply Project.toml]: `0.10.4`, or?

Comment (fredrikekre) [reply Project.toml]: Yea.

**Comment by KristofferC** [comment]
I'll do some maintenance of the way the docs are built and then tag a new version.

**Pull request merged**